app/error_controller.go

```diff
@@ -3,23 +3,25 @@ package app
 import (
 	"io"
 	"net/http"
+
+	"github.com/Arijeet-webonise/gameReview/pkg/framework"
 )
 
-func (app *App) Handler404Error(w http.ResponseWriter, r *http.Request) {
+func (app *App) Handler404Error(w *framework.Response, r *framework.Request) {
```

Comment:
why are you handling error in the `app`?Did not we write a change to the `framework` to have a global HTTP error handler?

Reply:
not using these since not using framework.Response.NotFound. But had to update since the framework change